alonzo/test/cddl-files/alonzo.cddl

```diff
@@ -13,6 +13,7 @@ block =
 transaction =
   [ transaction_body
   , transaction_witness_set
+  , bool
```

Comment:
I don't think this is actually correct because Alonzo is backwards compatible which means that wallet that didn't upgrade anything for Alonzo are still working. Transactions created by this wallet aren't adding this field and so I assume it means that the node actually accepts transactions that don't specific `isValid` in their transaction. Am I missing something maybe?

Reply:
I would need to check with the consensus folks to be sure, but I believe Mary transactions get translated to Alonzo by:

https://github.com/input-output-hk/cardano-ledger-specs/blob/0eb9b38bb125ec8eedf0f7aabf9e09f9b7bf79f7/alonzo/impl/src/Cardano/Ledger/Alonzo/Translation.hs#L106-L124